crates/octez/src/async/client.rs

```diff
@@ -163,14 +162,14 @@ impl TryFrom<StdOut> for Address {
 }
 
 #[derive(Debug)]
-pub struct OctezClient {
+pub struct AsyncOctezClient {
```

Comment:
Remove `Async` prefix for type, not needed as we are already in the `async` mod

Reply:
this will conflict w the `OctezClient` in octez crate

Reply:
You can use the sub-qualified name like `async::OctezClient` in the jstzd